interfaces/IBF-dashboard/src/app/types/ibf-layer.ts

```diff
@@ -106,7 +106,6 @@ export enum IbfLayerName {
   rainfall = 'rainfall',
   rainfallExtent = 'rainfall_extent',
   rainfallForecast = 'rainfall_forecast',
-  redCrescentBranches = 'red_crescent_branches',
```

Comment:
Are we removing support for Red Crescent branches because we removed Egypt?

Reply:
yes, that was a remnant of Egypt indeed